apps/saas/pages/index.tsx

```diff
@@ -2,16 +2,10 @@ import type { NextPage } from "next";
 import { useHomepageSet } from "../hooks/useHomepageSet";
 
 const Home: NextPage = () => {
-  const data = useHomepageSet();
-  const homepageItems = data.homepage?.content?.objects;
+  const { data } = useHomepageSet();
+  console.log("Homepage", data);
```

Comment:
console.log

Reply:
Ah this one, its just to stop errors on the homepage